ui/selectmenu.js

```diff
@@ -572,11 +576,12 @@ return $.widget( "ui.selectmenu", {
 	},
 
 	_toggleAttr: function() {
-		this.button
-			.toggleClass( "ui-corner-top", this.isOpen )
-			.toggleClass( "ui-corner-all", !this.isOpen )
-			.attr( "aria-expanded", this.isOpen );
-		this.menuWrap.toggleClass( "ui-selectmenu-open", this.isOpen );
+		this.button.attr( "aria-expanded", this.isOpen );
+
+		this._toggleClass( this.button, "ui-selectmenu-button-closed", null, !this.isOpen )
+			._toggleClass( this.button, "ui-selectmenu-button-open", null, this.isOpen )
```

Comment:
We see in the button pr these actually need to change to an `_remove/_add` because they are on the same element going to fix this here and add a comment.